components/org.wso2.carbon.status.dashboard.web/src/gadgets/chart-components/PieChart.jsx

```diff
@@ -95,64 +86,56 @@ export default class PieCharts extends React.Component {
 
                     });
                 }
-
                 data.map((datum) => {
                     randomUpdater++;
-                    let dataSetName = datum[colorIndex];
+                    const dataSetName = datum[colorIndex];
 
                     if (dataSets[dataSetName]) {
                         dataSets[dataSetName].y += datum[xIndex];
-
                     } else {
                         chartArray[0].colorIndex = chartArray[0].colorIndex >= chartArray[0].colorScale.length ? 0 : chartArray[0].colorIndex;
                         if (arcConfig.colorDomain) {
-                            let colorDomIndex = arcConfig.colorDomain.indexOf(dataSetName);
+                            const colorDomIndex = arcConfig.colorDomain.indexOf(dataSetName);
 
                             if (colorDomIndex > -1 && colorDomIndex < chartArray[0].colorScale.length) {
                                 dataSets[dataSetName] = {
                                     x: dataSetName,
                                     y: datum[xIndex],
-                                    fill: chartArray[0].colorScale[colorDomIndex]
+                                    fill: chartArray[0].colorScale[colorDomIndex],
                                 };
                                 chartArray[0].dataSetNames[dataSetName] = chartArray[0].colorIndex++;
                             } else {
                                 dataSets[dataSetName] = {
                                     x: dataSetName,
                                     y: datum[xIndex],
-                                    fill: chartArray[0].colorScale[chartArray[0].colorIndex]
+                                    fill: chartArray[0].colorScale[chartArray[0].colorIndex],
                                 };
                             }
-
                         } else {
                             dataSets[dataSetName] = {
                                 x: dataSetName,
                                 y: datum[xIndex],
-                                fill: chartArray[0].colorScale[chartArray[0].colorIndex]
+                                fill: chartArray[0].colorScale[chartArray[0].colorIndex],
```

Comment:
remove the last comma.. it is not necessary

Reply:
Fixed in bbb066d61f625e5e28b09b91ec2e5a2dc09ed919